tools/prepare_python_system_tests.py

```diff
@@ -11,7 +11,7 @@
                                  formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 parser.add_argument('-d', '--driver', required=True, type=str,
                     help='Python package name.',
-                    choices=os.listdir('src'))
+                    choices=['nidigital', 'nidmm', 'nidcpower', 'niscope', 'nifgen', 'nimodinst', 'nise', 'niswitch', 'nitclk'])
```

Comment:
Listing out the modules here would become a maintenance burden (when we add or remove modules). Can you obtain this list dynamically by parsing the data got from `https://api.github.com/repos/ni/nimi-python/releases`?

Reply:
Modified the 'prepare_python_system_test' to download the latest package first and then generating the choices list from the 'src' directory. Did the same in 'run_python_system_test'